### PR TITLE
feat: Added torchvision frontend

### PR DESCRIPTION
**Pull request by mohame54**
# PR Description

## Related Issue

Close #

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/__init__.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
```
Comment (AnnaTz) [inline ivy/functional/frontends/__init__.py:3]: Hey Mohammed, you have again committed changes to the demos folder😅 Please open a new PR without these so we can merge!

File: ivy_tests/test_ivy/test_frontends/test_torchvision/test_ops.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        ["float32", "float32"],
+        np.array(list(bbox.keys()), dtype=np.float32),
+        np.array(list(bbox.values()), dtype=np.float32),
+        iou_threshold,
+    )
+
+
+@st.composite
+def _roi_align_helper(draw):
+    dtype = draw(helpers.get_dtypes("float", full=False))[0]
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_torchvision/test_ops.py:38]: Is this function only supposed to work with `float`? If so, please add `@with_supported_dtypes` and use `get_dtypes("valid", ...` here. It's a matter of good practice.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_torchvision/test_ops.py (hunk 2 above)
Comment (mohame54) [reply ivy_tests/test_ivy/test_frontends/test_torchvision/test_ops.py:38]: I will change it.

File: ivy/functional/frontends/__init__.py (hunk 1 above)
Comment (mohame54) [reply ivy/functional/frontends/__init__.py:3]: okay, although I did not modify this folder

**Pull request closed without merging**